src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -149,21 +210,64 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
             lowContrast
           />
         )}
-        <Button
-          className="create-pipelinerun-button"
-          iconDescription="Create PipelineRun"
-          renderIcon={Add}
-          onClick={() => this.toggleModal(true)}
-        >
-          Create PipelineRun
-        </Button>
-        <CreatePipelineRun
-          open={this.state.showCreatePipelineRunModal}
-          onClose={() => this.toggleModal(false)}
-          onSuccess={this.handleCreatePipelineRunSuccess}
-          pipelineRef={pipelineName}
-          namespace={selectedNamespace}
-        />
+        {this.state.invalidFilter && (
+          <InlineNotification
+            lowContrast
+            kind="error"
+            title="Label filters must be of the format labelKey:labelValue"
+            subtitle=""
+            role="alert"
+          />
+        )}
+        <div className="search-bar">
+          <Search
+            placeHolderText="Input a label filter - labelKey:labelValue"
+            className="search"
+            labelText="Search by label filter"
+            onChange={this.handleChange}
+            value={this.state.currentFilterValue}
```

Comment:
Shouldn't reference state directly, unwrap from the state at the top of the component